src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/services/AssessmentNeedsService.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.assessrisksandneeds.services
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.AssessmentNeedDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.AssessmentNeedsDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.OffenderNeedDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.OffenderNeedsDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.restclient.AssessmentApiRestClient
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.services.exceptions.EntityNotFoundException
+
+@Service
+class AssessmentNeedsService(private val assessmentClient: AssessmentApiRestClient) {
+  fun getAssessmentNeeds(crn: String): AssessmentNeedsDto {
+    log.info("Get needs for CRN: $crn")
+    val offenderNeeds = assessmentClient.getNeedsForCompletedLastYearAssessment(crn)
+    if (offenderNeeds?.historicStatus != "CURRENT") {
```

Comment:
let's define constants instead of hardcoding them

Reply:
Done :)